pyproject.toml

```diff
@@ -1,54 +1,197 @@
 [tool.black]
-    line-length = 88
-    target-version = ['py36', 'py37']
-    exclude = '''
-    (
-      /(
-          \.eggs
-        | \.git
-        | \.pytype
-        | \.pytest_cache
-        | build
-        | dist
-      )/
-    )
-    '''
+line-length = 88
+target-version = [ "py36", "py37",]
+exclude = "((.eggs | .git | .pytype | .pytest_cache | build | dist))"
+
+[tool.poetry]
+name = "rasa"
+version = "1.7.0"
+description = "Open source machine learning framework to automate text- and voice-based conversations: NLU, dialogue management, connect to Slack, Facebook, and more - Create chatbots and voice assistants"
+authors = [ "Rasa Technologies GmbH <hi@rasa.com>",]
+maintainers = [ "Tom Bocklisch <tom@rasa.com>",]
+homepage = "https://rasa.com"
+repository = "https://github.com/rasahq/rasa"
+documentation = "https://rasa.com/docs"
+classifiers = [ "Development Status :: 4 - Beta", "Intended Audience :: Developers", "License :: OSI Approved :: Apache Software License", "Programming Language :: Python", "Programming Language :: Python :: 3.6", "Programming Language :: Python :: 3.7", "Topic :: Software Development :: Libraries",]
+keywords = [ "nlp", "machine-learning", "machine-learning-library", "bot", "bots", "botkit", "rasa conversational-agents", "conversational-ai", "chatbot", "chatbot-framework", "bot-framework",]
+include = [ "LICENSE.txt", "README.md", "rasa/core/schemas/*", "rasa/core/training/visualization.html", "rasa/nlu/schemas/*", "rasa/cli/default_config.yml",]
```

Comment:
what about the initial `scaffold` project?

Reply:
This section was simply copied from `setup.cfg`

Reply:
Mhm, in the `setup.py` we make sure it's included by doing `include_package_data`. Can you please verify that it works correctly?

Reply:
I've build a package and the zip contains the initial project, so should be fine